docs/ibexa_cloud/ibexa_cloud_guide.md

```diff
@@ -51,7 +51,7 @@ All you have to do is access it and you can start deploying resources and develo
 
 ### Performance management
 
-Make your applications more effective, scalable, and effective by using the [Observability Suite](https://platform.sh/features/observability-suite/).
+Make your applications more effective, scalable, and effective by using the [Observability Suite](https://upsun.com/product/).
```

Comment:
I found no equivalent to https://web.archive.org/web/20250905005602/https://platform.sh/features/observability-suite/
I would go with https://fixed.docs.upsun.com/increase-observability/integrate-observability.html or its parent.
I'll try to check with Upsun.